tests/apps/courses/test_templates_course_detail.py

```diff
@@ -41,62 +45,66 @@ def test_course_cms_published_content(self):
         # The page should not be visible before it is published
         url = page.get_absolute_url()
         response = self.client.get(url)
-        self.assertEqual(response.status_code, 404)
```

Comment:
why remove this test?

Reply:
That was an error from mine for unknow reason.